uReplicator-Controller/src/main/java/com/uber/stream/kafka/mirrormaker/controller/core/ManagerControllerHelix.java

```diff
@@ -68,6 +71,14 @@ public synchronized void start() {
         new ControllerStateModelFactory(this));
     try {
       _helixZkManager.connect();
+      ZkHelixPropertyStore<ZNRecord> propertyStore = _helixZkManager.getHelixPropertyStore();
+      String resourcePath = Constants.PARTICIPATE_INSTANCE_ID_HOSTNAME_PROPERTY_KEY;
+      ZNRecord znRecord = propertyStore.get(resourcePath, null, AccessOption.PERSISTENT);
+      if (znRecord == null) {
+        znRecord = new ZNRecord(resourcePath);
+      }
+      znRecord.setSimpleField(_instanceId, _hostname);
```

Comment:
Can we set top level as something like "Controller id to hostname map"? We might need this map for worker as well. Also we might use propertyStore for other purpose in the future.